10-epr-enhanced/00-intro.md

```diff
@@ -0,0 +1,47 @@
+## EPR Enhanced
+
+## Overview
+
+In this section we will expand on our EPR concepts we have learned in the
+previous sections. This is a free form exercise to expand on what we have
+learned.
+
+## Watcher
+
+Add a watcher to the EPR system using the watcher SDK. Expand on the example
+from the previous section.
+
+Suggestions:
+
+- Add a watcher to fire a webhook.
+
+## CDEvents
+
+Add CDEvents to the EPR system and expand on the example from the previous
+section.
+
+Suggestions:
+
+- Add a watcher to create a CDEvent.
+
+## SBOMs
+
+Add SBOMs to the EPR system and expand on the example from the previous section.
+
+Suggestions:
+
+- Add a watcher to create a SBOM.
+
+## Python
+
+Add Python to the EPR system and expand on the example from the previous
+section.
+
+Suggestions:
+
+- Add a Python watcher to create an Event.
```

Comment:
Is this not redundant with the previous "Watcher" entry? Should one be in Go and another in Python?

Reply:
The idea with this section is to have participants use knowledge from previous sessions to do something on their own.